### ISSUE TITLE
Directory path does not exist in Plex docker

### ISSUE THREAD

**Issue by tarunkumar519** [I-am-PUID-0/pd_zurg#51]
**Describe the bug**
Rclone RD process not running when system reboots or in general

**To Reproduce**
i already had plex setup, real-debrid for api then followed instructons here [https://docs.google.com/document/d/1wGzoV11vG2jiSiKm_EoShrcig70pWYPDYiYbAXT5jRQ/edit#heading=h.nu4h35dxqc4q](url)

**Expected behavior**
Continuous operation of underlying scripts

**Screenshots**

Please see attached logs
[pd_zurg_logs.txt](https://github.com/I-am-PUID-0/pd_zurg/files/15211856/pd_zurg_logs.txt)

Please complete the following information:
    OS: Debian 12, Docker version 26.1.1, build 4cf5afa, Portainer for stack staging

**Comment by joobert** [comment]
Thought I was going crazy, I'm having this exact same issue

**Comment by joobert** [comment]
Are you seriously gonna close the issue without telling what you did to fix??

**Comment by tarunkumar519** [comment]
Heyy, sorry about that. I thought i deleted or closed my issue long ago. The issue question and what problem I had was different (when i posted the issue question it got resolved- so i changed the content to the next problem i had. )Sorry for the confusion.

I thought maybe you will open a new issue with exactly what problem you have. But here are my solutions: 


1. If you had the Plex directory path does not exist error - it means you maybe running it in docker (both plex and pd_zurg) , the PLEX DIR PATH env variable should be the container name of volume (not hostname) of zurg directory for Plex.

for example:
Plex volume: /zurg is container name & the binded host volume is something like /mnt/pd_zurg where your movies & shows are linked by real debrid.

so the PLEX DIR PATH variable is /zurg

2. if you had the RD quitting error, i donno how i fixed it, but when I fixed my dir path error i just removed all the files and freshly installed everything again and it just worked.
Hope this helped
